pkgs/development/node-packages/node-packages.nix

```diff
@@ -76678,112 +76417,6 @@ in
     bypassCache = true;
     reconstructLock = true;
   };
-  hsd = nodeEnv.buildNodePackage {
```

Comment:
When an conflict occurs in `node-packages.nix`, you'll need to re-run `generate.sh` again. Otherwise, recent changes made in the `master` branch would be deleted wholesale.

Reply:
Oh, my bad. I incorrectly assumed that only a package update was run but there were some additions I missed while checking the commit history.

Reply:
As far as I'm aware, updates to node packages are rarely pure updates. People keep adding stuff to `nodePackages` frequently enough that it's up to date most of the time ;)